app/src/main/java/bacon/john/bitcoinpricewatch/bitcoin/BitcoinCurrencyManager.java

```diff
@@ -0,0 +1,120 @@
+package bacon.john.bitcoinpricewatch.bitcoin;
+
+import org.json.JSONObject;
+
+import java.util.*;
+import org.json.*;
+import java.text.*;
+import io.realm.*;
+
+import bacon.john.bitcoinpricewatch.localstorage.LocalStorageService;
+
+/**
+ * @class BitcoinCurrencyManager
+ *
+ * Retrieves information about bitcoin in various currencies
+ */
+public class BitcoinCurrencyManager {
+    public interface CurrencyListCallback {
```

Comment:
not totally sure on callback best practices, but this felt very natural and was easy to consume; perhaps it is too iOS-like though